ksqldb-rest-model/src/main/java/io/confluent/ksql/rest/entity/SimpleConnectorPluginInfo.java

```diff
@@ -33,7 +32,7 @@
 
   @JsonCreator
   public SimpleConnectorPluginInfo(
-      @JsonProperty("className") final String className,
+      @JsonProperty("class") final String className,
```

Comment:
This change is backwards incompatible, right? This class is returned from the public API response for `DESCRIBE CONNECTOR PLUGINS;`. I don't think we can make a change like this without bumping the API version.

What's the motivation for this change?